Library/Homebrew/global.rb

```diff
@@ -47,8 +47,8 @@
 HOMEBREW_USER_AGENT_RUBY =
   "#{ENV.fetch("HOMEBREW_USER_AGENT")} ruby/#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}"
 HOMEBREW_USER_AGENT_FAKE_SAFARI =
-  "Mozilla/5.0 (Macintosh; Intel Mac OS X 13_5_1) AppleWebKit/605.1.15 " \
-  "(KHTML, like Gecko) Version/16.5 Safari/605.1.15"
+  "Mozilla/5.0 (Macintosh; Intel Mac OS X 14_0) AppleWebKit/605.1.15 " \
```

Comment:
Does Safari output this? I thought they froze it at `10_15_7`

Reply:
@Bo98 You are right, it does not.